user-service/templates/accessrule.yaml

```diff
@@ -1,12 +1,11 @@
 apiVersion: oathkeeper.ory.sh/v1alpha1
 kind: Rule
 metadata:
-  name: user-service-users
-  namespace: user-service
+  name: {{ include "user-service.fullname" . }}
 spec:
   match:
-    methods: ["GET","PUT","DELETE","POST"]
-    url: "https://users.api.dev.mindtastiuc.lol/user/<.+>"
+    methods: {{ .Values.matchMethods }}
+    url: {{ .Values.matchPath }}
   authenticators:
     - handler: bearer_token
       config:
```

Comment:
Since kratos and koda are stated explicitly under `config:` here. Would be nice to have application dependencies in argo. But that seems to be far away (argoproj/argo-cd#3517). Are there any other best practices for handling such dependencies?

Reply:
Not that I know other than documenting these things, unfortunately. They could be exposed to Values.yaml though, that's right.